openvdb_houdini/CMakeLists.txt

```diff
@@ -119,12 +120,15 @@ endif()
 set(OPENVDB_HOUDINI_INSTALL_PREFIX ${HOUDINI_INSTALL_LOCATION}
   CACHE PATH "Base install path for OpenVDB Houdini shared libraries, dsos and headers."
 )
+set(OPENVDB_HOUDINI_CONFIGDIR "${OPENVDB_HOUDINI_INSTALL_PREFIX}/config")
+set(OPENVDB_HOUDINI_DSODIR "${OPENVDB_HOUDINI_INSTALL_PREFIX}/dso")
+set(OPENVDB_HOUDINI_INCLUDEDIR "${OPENVDB_HOUDINI_INSTALL_PREFIX}/${CMAKE_INSTALL_INCLUDEDIR}")
+set(OPENVDB_HOUDINI_LIBDIR "${OPENVDB_HOUDINI_INSTALL_PREFIX}/${CMAKE_INSTALL_LIBDIR}")
+set(OPENVDB_HOUDINI_PYTHONRC_DIR "${OPENVDB_HOUDINI_INSTALL_PREFIX}/python2.7libs")
+set(OPENVDB_HOUDINI_SCRIPTSDIR "${OPENVDB_HOUDINI_INSTALL_PREFIX}/scripts")
```

Comment:
If we're to set these (which I think we 100% should) we should consider exposing the first folder item as cache variables. Lost of studios have different ways they are deploying Houdini/its plugins, I'm already symlinking stuff from the current vdb builder folders to get things working here

Reply:
Ditto -- would the "first folder item" be the existing `OPENVDB_HOUDINI_INSTALL_PREFIX`?

Would it be okay to make all of these cache variables?  It seems like we should.  Pardon my cmake ignorance -- can any variable be overriden via `cmake -D FOO=BAR` or can only cache variables be overridden?  If only cache variables, then these should definitely be made into cache variables.  I'll go ahead and make that tweak shortly and push it up while I wait to hear back about the other questions.

Reply:
> can any variable be overriden via cmake -D FOO=BAR or can only cache variables be overridden?

It's possible to have two of the same variable at any given time. For example, with a CMake variable `MY_VAR`, you can have a "normal" (for lack of better CMake terminology) version of the variable and a `CACHE` version of the variable. The "normal" version will take precedence unless you request the `CACHE` version. I believe that whenever you specify a variable from the command line, it's inserted as a `CACHE` variable:
```shell
cmake -DMY_VAR="bar" ../
```

```cmake
...
# MY_VAR is fetched from the CACHE as no local variable has been set
message(STATUS ${MY_VAR}) # prints bar
# This creates a new local MY_VAR and sets it to "foo". The cache is untouched
set(MY_VAR "foo")
# MY_VAR exists as a local which is prioritised
message(STATUS ${MY_VAR}) # prints foo
# set() sees the CACHE keyword so tries to set the CACHE version of 
# MY_VAR to "cached". However, as MY_VAR already contains a value it is 
# left untouched
set(MY_VAR "cached" CACHE STRING "some help text" )
# The above set() command forces CMake to drop all local values of MY_VAR
# so "bar" is printed... :|
message(STATUS ${MY_VAR}) # prints bar
...
```

I can't find up to date documentation on this, but see the * Variable types in CMake * section on the below link which summarises it pretty well in one section. 

https://cmake.org/cmake/help/v3.0/command/set.html